app/src/main/java/ca/chronofit/chrono/circuit/CircuitDashboardFrag.kt

```diff
@@ -152,6 +166,16 @@ class CircuitDashboardFrag : Fragment() {
         intent.putExtra("lastRest", lastRest)
         intent.putExtra("soundEffect", soundEffect)
         startActivityForResult(intent, Constants.DASH_TO_TIMER)
+
+        val index = circuitsObject.circuits.indexOf(circuit)
+        val currCount = circuitsObject.circuits[index].count
+        if (currCount != null) {
+            circuitsObject.circuits[index].count = currCount + 1
+        } else {
+            circuitsObject.circuits[index].count = 0
```

Comment:
We would want this to be 1. Once the object is created this value should be 0, but in this case, it will be null, so when the click it for the first time, it should go to 1, not zero. Minor technicalities, but makes it more readable imo.

Reply:
Makes sense, done.